manifests/params.pp

```diff
@@ -104,8 +99,6 @@
 
       if $postgresql::globals::postgis_package_name {
         $postgis_package_name = $postgresql::globals::postgis_package_name
-      } elsif $facts['os']['release']['major'] == '5' {
-        $postgis_package_name = 'postgis'
       } elsif $postgis_version and versioncmp($postgis_version, '2') < 0 {
```

Comment:
Is there still PostGIS 1 support for Red Hat distros? Feels like this could be simplified.

Reply:
I have zero knowledge about postgis so I didn't want to touch it

Reply:
ack, I dropped it